src/components/Typography/Typography.module.css

```diff
@@ -0,0 +1,94 @@
+.h1 {
+  font-size: var(--rvr-font-size-h1);
```

Comment:
Minor: I'd suggest dropping these `.h*` helper classes and the associated `--rvr-font-size-*` variables.

The way this is currently implemented, the `.h1` style will only be accessible by passing it in as a `className` prop. Since we're using CSS modules, that would require consumers to look at the components implementation to get the right classname.

I'd guess that developers are more likely to pick sizes and colors straight from Figma and use the `size` and `color` props.

If you _really_ want shortcuts for things like `h1`, you could do...
```
export const H1 = (props) => <Typography size="xl3" tag="h1" {...props} />
```

Let me know if I'm missing something.